extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelRuntimeCatalog.java

```diff
@@ -28,7 +28,8 @@ public CamelRuntimeCatalog(CamelConfig.RuntimeCatalogConfig config) {
     @Override
     public String modelJSonSchema(String name) {
         if (!config.models) {
-            return null;
+            throw new RuntimeException(
```

Comment:
We need to check if throwing an exception is the right thing, I do not remember if there is any assumption in the code path that invokes this method

Reply:
I am a troll, I have put it there to see who and how is actually using it :) I have fixed the tests in fc0bdf6. Are you using it in Camel K?